src/Platform/Microsoft.Testing.Platform/OutputDevice/IPlatformOutputDevice.cs

```diff
@@ -14,7 +13,7 @@ internal interface IPlatformOutputDevice : IExtension
 
     Task DisplayAfterSessionEndRunAsync();
 
-    Task DisplayAsync(IOutputDeviceDataProducer producer, IOutputDeviceData data);
+    Task DisplayAsync(IOutputDeviceData data);
```

Comment:
While `IPlatformOutputDevice` is internal today. I think we will want to expose `IOutputDeviceDataProducer` if we will make it public, even if it's unused in today's scenarios. I recall having some discussion around this topic few months back with @Evangelink and @MarcoRossignoli.